joint_state_publisher/joint_state_publisher/joint_state_publisher.py

```diff
@@ -67,24 +67,23 @@ def init_sdf(self, robot):
                 continue
             if child.localName == 'joint':
                 jtype = child.getAttribute('type')
-                if jtype in ['gearbox', 'revolute2', 'ball', 'screw', 'universal', 'fixed']:
+                if jtype in ('gearbox', 'revolute', 'ball', 'screw', 'universal', 'fixed'):
```

Comment:
Not sure why `revolute2` is changed to `revolute`. I believe `revolute` is in the list of supported joint types (https://github.com/ros/sdformat_urdf/blob/ros2/sdformat_urdf/README.md#joint-limitations)